src/component/menu/routes.js

```diff
@@ -95,7 +95,7 @@ export const routes = [
         menu: {},
     },
     {
-        path: '/projects/:projectId/features/:featureId/settings',
+        path: '/projects/:projectId/features/:featureId/edit',
```

Comment:
Should the edit description link on the feature overview page go to the edit page? :thinking:

Reply:
I think so because if we take him to the settings tab it will be an extra step (click on Edit Metadata) instead of going directly to the edit page.